apps/fluent-tester/src/TestComponents/Badge/E2EBadgeTest.tsx

```diff
@@ -9,7 +9,7 @@ export const E2EBadgeTest: React.FunctionComponent = () => {
       <Badge testID={BADGE_TEST_COMPONENT} appearance="outline" size="extraLarge" color="success" shape="square">
         Basic Badge
       </Badge>
-      <Badge testID={BADGE_SECONDARY_TEST_COMPONENT}>Default Badge</Badge>
+      <PresenceBadge testID={BADGE_SECONDARY_TEST_COMPONENT} status="available" outOfOffice={true} />
```

Comment:
Does this need to be gated via an SVG check?

Reply:
(Probably the better way to handle this is to have the component return null on platforms that don't support SVG, but that'll be kind of moot soon since eventually SVGs should be on RNW.)